README.md

```diff
@@ -36,7 +36,44 @@ Fedimint UI releases use semantic versioning (`major.minor.patch`)
 
 ## Running a Release
 
-### Build and Run from Source
+### Option 1 - Running with Nix (preferred)
+
+1. Install Nix
+   ```bash
+   curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+   ```
+1. Add `experimental-features = nix-command flakes` to your `/etc/nix/nix.conf` file   
+1. In terminal `cd` to the fedimint-ui repo root directory
+1. Enter the following command to start Nix development environment
+   ```bash
+   nix develop
+   ```
+1. Run `yarn nix-gateway` or `yarn nix-guardian` Note: **nix-gateway** preconfigures the federation so you don't have to go through federation setup. **nix-guardian** starts separate guardian nodes that are connected into the federation when you run through the federation setup process.
+
+#### **Gateway UI**
+```bash
+yarn nix-gateway
+```
+nix-gateway spins up these instances
+| instance | url | 
+| ---------- | --- |
+| gateway-ui | http://127.0.0.1:3004/ |
+| guardian-ui | http://127.0.0.1:3000/ |
+
+#### **Guardian UI**
+```bash
+yarn nix-guardian
+```
+nix-guardian spins up these instances
+| instance | url |
+| ------------- | --- |
+| guardian-ui-1 | http://127.0.0.1:3000/ |
+| guardian-ui-2 | http://127.0.0.1:3001/ |
+| guardian-ui-3 | http://127.0.0.1:3002/ |
+| guardian-ui-4 | http://127.0.0.1:3003/ |
+
+
+### Option 2 - Build and Run from Source
```

Comment:
Can we make this the third option? IMO it's the most tedious to use

Reply:
Wait I think this is mistaken, the running with nix is meant for development, build and run from source and run with docker below are meant for running releases. The nix instructions should be under the `## Development` header on line 133, not under running a release.

Reply:
My bad, I moved them down into ## Development and reverted everything up in ## Running a release. Ready for review